core/trino-spi/src/main/java/io/trino/spi/eventlistener/ColumnMaskReferenceInfo.java

```diff
@@ -13,27 +13,19 @@
  */
 package io.trino.spi.eventlistener;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
+import static java.util.Objects.requireNonNull;
 
 /**
  * This class is JSON serializable for convenience and serialization compatibility is not guaranteed across versions.
  */
-public class ColumnMaskReferenceInfo
-        extends FilterMaskReferenceInfo
+public record ColumnMaskReferenceInfo(@Override String expression, @Override String targetCatalogName, @Override String targetSchemaName, @Override String targetTableName, String targetColumnName)
```

Comment:
What does `@Override` do here? I've never seen that and can't find any reference for it

Reply:
It’s annotating the field accessor method. The details are a little esoteric, but are covered in the JEP for record classes: https://openjdk.org/jeps/395